spec/runners/unified/ddl_operations.rb

```diff
@@ -32,7 +39,7 @@ def drop_collection(op)
     def assert_collection_exists(op, state = true)
       consume_test_runner(op)
       use_arguments(op) do |args|
-        client = ClientRegistry.instance.global_client('authorized')
+        client = ClientRegistry.instance.global_client('root_authorized')
```

Comment:
I am not sure if this is the right way to fix the issue, but without this change specs fail with
```
[13:Unauthorized]: not authorized on ts-tests to execute command { listCollections: 1, cursor: {}, nameOnly: true, $db: "ts-tests", lsid: { id: UUID("4e1557e6-2023-4eba-98e4-ce33bd6e99e4") } } (on localhost:27017, modern retry, attempt 1)
```

Reply:
This is probably a-ok, the biggest issue should be that  root_authorized doesn't share cluster with authorized hence would create more overhead.

The databases/collections used in spec tests are enumerated in https://github.com/mongodb/mongo-ruby-driver/blob/master/spec/support/spec_config.rb#L554 for access control purposes.

Reply:
Actually, I added `ts-tests` to spec_config - https://github.com/mongodb/mongo-ruby-driver/pull/2293/files#diff-88804426c739314d026c14eaa170a02831dd50c15481e2125d93fe74818ccb8bR596

Still failing without root_authorized.